_visualisation/core/parties-nodes.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Parties (Nodes)
+layout: visualisation
+---
+
+Nodes represent interested Parties, such as Persons, Entities or Regimes.
+{: .lead}
+
+
+## Appearance
+
+* Nodes are drawn as a circle containing an icon.
+  * Always draw circles/icons at the same size.
+  * Try to space nodes at regular intervals across the diagram.
+* The icon used indicates the type of Party that the node represents.
+  * See the [Vocabulary](/visualisation/core/vocabulary) for the valid types of Party and the correct icons to use for each.
+
+
+## Labels
+
+Nodes must have a Label underneath, identifying the Party. The Label can either be:
+
+* The Party's proper name (if known).
+* A unique Identifier that you create (see below).
+
+
+### Identifiers
+
+Instead of naming a Party in its Label, you can create an Identifier. Do this when identifying the name is not [relevant](/visualisation/core/relevance) or is not known. An Identifier is formed from:
+
+|      | Status +       | Type +             | Unique ID
+|      | (if required)  | The type of Party  | A unique incrementing letter or number
+| --------------------------------------------
+| e.g. |                | Entity             | A
```

Comment:
No, it's an example of an Identifier where there is no Status to note. Realistically I think this only occurs if the Parties are fictional, as otherwise yes you ought to note that they are either Unknown or Anonymous. Does that make sense? Should I clarify this?

Reply:
Ah I see, I think again this is just from me reading too much into a table. Maybe just writing these examples out would be clearer?